src/main/resources/logback.xml

```diff
@@ -21,8 +21,23 @@
         </encoder>
     </appender>
 
+    <appender name="DISCORD" class="com.github.napstr.logback.DiscordAppender">
+        <webhookUri>https://discordapp.com/api/webhooks/937000047492844289/yoUKnOwwHAtTHiSiSFaKe</webhookUri>
```

Comment:
Would it be possible to not set this or leave it blank? `DiscordAppender` should then wait until it is set before trying to send messages, see:
https://github.com/napstr/logback-discord-appender/blob/a653e826872adb354d2180066ccc1812bca39ac8/src/main/java/com/github/napstr/logback/DiscordAppender.java#L57

Reply:
thats what im asking in https://github.com/mojira/arisa-kt/pull/350#discussion_r448792562